test/framework/shootcreationframework.go

```diff
@@ -312,7 +312,7 @@ func RegisterShootCreationFrameworkFlags() *ShootCreationConfig {
 	flag.StringVar(&newCfg.shootMachineType, "machine-type", "", "the Machine type of the first worker of the test shoot. Needs to match the machine types for that Provider available in the CloudProfile.")
 	flag.StringVar(&newCfg.shootMachineImageVersion, "machine-image-version", "", "the Machine Image version of the first worker of the test shoot. Needs to be set when the MachineImageName is set.")
 	flag.StringVar(&newCfg.cloudProfileName, "cloud-profile-name", "", "cloudProfile name to use for the shoot.")
-	flag.StringVar(&newCfg.cloudProfileKind, "cloud-profile-kind", "", "cloudProfile kind to use for the shoot. Optional.")
+	flag.StringVar(&newCfg.cloudProfileKind, "cloud-profile-kind", "CloudProfile", "cloudProfile kind to use for the shoot. Optional.")
```

Comment:
```suggestion
	flag.StringVar(&newCfg.cloudProfileKind, "cloud-profile-kind", v1beta1constants.CloudProfileReferenceKindCloudProfile, "cloudProfile kind to use for the shoot. Optional.")
```
You could use the constant.